src/Microsoft.IdentityModel.Tokens/project.json

```diff
@@ -3,10 +3,10 @@
   "description": "Includes types that provide support for cryptographic operations.",
   "dependencies": {
     "Microsoft.IdentityModel.Logging": "1.0.0-*",
-    "Newtonsoft.Json": "6.0.6"
+    "Newtonsoft.Json": "8.0.3"
```

Comment:
nit: IIRC, the ASP.NET team mainly uses JSON.NET 8.0.2. Maybe you should ping them so they eventually migrate to the latest bits before RC2 is released.